### ISSUE TITLE
Example sklearn-onnx code in documentation does not work.

### ISSUE THREAD

**Issue by davidrs** [onnx/sklearn-onnx#955]
Current example provided at https://onnx.ai/sklearn-onnx/

Fails on the first inference line due to:
```
ValueError: This ORT build has ['TensorrtExecutionProvider', 'CUDAExecutionProvider', 'CPUExecutionProvider'] enabled. Since ORT 1.9, you are required to explicitly set the providers parameter when instantiating InferenceSession. For example, onnxruntime.InferenceSession(..., providers=['TensorrtExecutionProvider', 'CUDAExecutionProvider', 'CPUExecutionProvider'], ...)
```

I believe it is because `sess = rt.InferenceSession("rf_iris.onnx")` is missing some equivalent of: `sess = rt.InferenceSession("rf_iris.onnx", providers=['CPUExecutionProvider'])`

NOTE: I believe that will just set a CPU provider,  I also tried: ` providers=rt.get_available_providers()` but since I'm on a TensorRT + CUDA device it threw errors. I'm not sure if that's because it is not supported or requires different params if on GPU.
```
RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization: /onnxruntime_src/onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc:867 SubGraphCollection_t onnxruntime::TensorrtExecutionProvider::GetSupportedList(SubGraphCollection_t, int, int, const onnxruntime::GraphViewer&, bool*) const [ONNXRuntimeError] : 1 : FAIL : TensorRT input: label has no shape specified. Please run shape inference on the onnx model first. Details can be found in https://onnxruntime.ai/docs/execution-providers/TensorRT-ExecutionProvider.html#shape-inference-for-tensorrt-subgraphs
```

tl;dr Current example fails and just needs `, providers=['CPUExecutionProvider']` to work on CPU. Unclear from example if/how it should be setup on GPU. I _think_ there is no special GPU support for these  models, but maybe I'm wrong.

**Comment by 876399730** [comment]
i met the same question.if you solve it,please reply me .thanks!
